h/admin/views/cohorts.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.view import view_config
+from pyramid import httpexceptions as exc
+
+from h import models
+from h import paginator
+
+
+@view_config(route_name='admin_cohorts',
+             request_method='GET',
+             renderer='h:templates/admin/cohorts.html.jinja2',
+             permission='admin_cohorts')
```

Comment:
The permission here should probably just be `admin_feature`, not `admin_cohorts` -- you can either admin features or you can't -- there's no need to be more fine-grained than that.